### PR TITLE
Add read view pagination

### LINKED ISSUE #4281: Read views: support for functional indexes
**Product:** Tarantool
**SME:** @ Gumix

# Details

> <…>the following limitations:
> 
> Only the memtx engine is supported.
> **|Only TREE and HASH indexes are supported.|**
> Pagination is not supported (the after option).
> 
> 
> 
> Working with<…>

https://www.tarantool.io/en/doc/latest/enterprise/read_views/#limitations

Functional indexes are also supported

### PR DESCRIPTION
**Pull request by p7nov**
Resolves https://github.com/tarantool/enterprise_doc/issues/218, #4281 

[Read views](https://docs.d.tarantool.io/en/doc/gh-ee-218-read-view-pagination/enterprise/read_views/) doc page:
- Add pagination example
- Add functional indexes to the list of supported index types

Deployment: https://docs.d.tarantool.io/en/doc/gh-ee-218-read-view-pagination/enterprise/read_views/

### CODE REVIEW

File: doc/enterprise/read_views.rst [hunk 1]
```diff
@@ -104,6 +104,39 @@ Similarly, you can retrieve data by the specific index.
...
+.. code-block:: tarantoolsession
+
+    tarantool> read_view1.space.bands:select({}, { limit = 3, after = position })
+    ---
+    - - [4, 'The Beatles', 1960]
+      - [5, 'Pink Floyd', 1965]
+      - [6, 'The Rolling Stones', 1962]
+    ...
 
 ADD example with fech_pos and after
```
Comment (drewdzzz) [inline doc/enterprise/read_views.rst]: Seems irrelevant - the patch introduces such example.

**Review by drewdzzz — APPROVED** [review]

File: doc/enterprise/read_views.rst (hunk 1 above)
Comment (p7nov) [reply doc/enterprise/read_views.rst]: thanks, that's a bug. Fixed

**Review by andreyaksenov — APPROVED** [review]
